### PR TITLE
Tested NO_IMAGE_URI for non-emptiness instead

### PR DESCRIPTION
**Pull request by yomexzo**
Implements the comment I missed on #127.

👀 @minasmart, @tessalt

### CODE REVIEW

File: tests/unit/models/product-model-test.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ test('it attaches a reference to the config on variants', function (assert) {
 test('it returns Shopify admin\'s no image URI', function (assert) {
   assert.expect(1);
 
-  assert.equal(NO_IMAGE_URI, 'https://widgets.shopifyapps.com/assets/no-image.svg');
+  assert.equal(true, typeof NO_IMAGE_URI === 'string' && NO_IMAGE_URI.length !== 0, 'NO_IMAGE_URI must be a non-empty string');
```
Comment (tessalt) [inline tests/unit/models/product-model-test.js]: i think you just want `assert.ok` here with the expression that should be truthy.

Comment (minasmart) [inline tests/unit/models/product-model-test.js]: It's also two assertions.

Comment (yomexzo) [inline tests/unit/models/product-model-test.js]: oh. Thanks!

Comment (yomexzo) [inline tests/unit/models/product-model-test.js]: updated @minasmart, @tessalt

File: tests/unit/models/product-model-test.js [hunk 2]
```diff
@@ -97,9 +97,10 @@ test('it attaches a reference to the config on variants', function (assert) {
 });
 
 test('it returns Shopify admin\'s no image URI', function (assert) {
-  assert.expect(1);
+  assert.expect(2);
 
-  assert.equal(NO_IMAGE_URI, 'https://widgets.shopifyapps.com/assets/no-image.svg');
+  assert.ok(typeof NO_IMAGE_URI === 'string', 'NO_IMAGE_URI must be a string');
+  assert.ok(NO_IMAGE_URI.length !== 0, 'NO_IMAGE_URI must NOT be empty');
```
Comment (minasmart) [inline tests/unit/models/product-model-test.js]: `assert.ok(NO_IMAGE_URI.length` should be ok. non-zero is truthy.

Comment (yomexzo) [inline tests/unit/models/product-model-test.js]: true. updated now.

**Comment by minasmart** [comment]
Looks good to me! Just squash those commits 👍

**Comment by yomexzo** [comment]
hey @tessalt .. would appreciate a second 👀 before merging.

Thanks!

**Comment by tessalt** [comment]
LGTM

**Pull request merged**